core/store/src/migrations.rs

```diff
@@ -138,7 +138,7 @@ pub fn migrate_6_to_7(path: &Path) {
 }
 
 pub fn migrate_7_to_8(path: &Path) {
-    let store = create_store(path);
+    let store = create_store(path, None);
```

Comment:
Instead of adding a ton of `, None`, I would introduce a new `create_store_with_settings` method in addition to `create_store` and leave the old code as is, and the new code can pass db_settings explicitly without a need to wrap it into `Some`

Reply:
@frol Yeah, after I wrote the change I realized I don't like it either.